packages/system/talos-log-collector/README.md

```
@@ -0,0 +1,96 @@
+      - endpoint: "tcp://127.0.0.1:5170/"
+        format: "json_lines"
+---
+apiVersion: v1alpha1
+kind: KmsgLogConfig
+name: talos-log-collector
+url: "tcp://127.0.0.1:5170/"
+```
+
+> Omitting the `KmsgLogConfig` document collects service logs only; kernel
+> (kmsg) logs, including the DRBD messages, will silently never arrive.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)